test/unit/codegen/llvm.cpp

```diff
@@ -202,9 +202,14 @@ SCENARIO("Unary expression", "[visitor][llvm]") {
             std::smatch m;
 
             std::regex allocation(R"(%1 = load double, double\* %a)");
-            std::regex negation(R"(fneg double %1)");
             REQUIRE(std::regex_search(module_string, m, allocation));
-            REQUIRE(std::regex_search(module_string, m, negation));
+
+            // llvm v9 and llvm v11 implementation for negation
```

Comment:
Good catch! That's actually quite important - is there a specific version of LLVM that we target (I was thinking of the most recent one), or do we want to support a multiple of these?

Reply:
I think it would be fine to target last one or two. It's just that on our Jenkins CI node we have 9.0 now and hence test were failing. So I changed this. 

In Jan we will switch to LLVM 11 on our CI nodes as well. So we can just target latest one and it would be sufficient.

Reply:
Okay, that makes sense!